src/unit_tests/os_crypto/sha1/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ cmake_minimum_required(VERSION 3.10)
 list(APPEND os_crypto_sha1_tests_names "test_sha1_op")
 list(APPEND os_crypto_sha1_tests_flags "-Wl,--wrap,fopen -Wl,--wrap,fclose -Wl,--wrap,fflush -Wl,--wrap,fread \
                                         -Wl,--wrap,fseek -Wl,--wrap,fwrite -Wl,--wrap,remove -Wl,--wrap,fprintf \
-                                        -Wl,--wrap,fgets -Wl,--wrap,fgetpos -Wl,--wrap,fgetc")
+                                        -Wl,--wrap,fgets -Wl,--wrap,fgetpos -Wl,--wrap,fgetc -Wl,--wrap,popen")
```

Comment:
Is it necessary to add this wrap?

Reply:
Yes, it is necessary because the `__wrap_popen` mock was modified to use test_mode, and it must be added so that it finds `__real_popen`

Reply:
Ditto

Reply:
As discussed in the huddle, it was determined that they were necessary